src/nxdoc/nuxeo-add-ons/nuxeo-retention-management/nuxeo-retention-howto-retain-multiple-blobs.md

```diff
@@ -0,0 +1,240 @@
+---
+title: 'HOWTO: Retain Multiple Files'
+description: Discover how to retain multiple blobs (files) and prevent properties from being edited when a record is declared.
+review:
+    comment: ''
+    date: '2024-01-22'
+    status: 'ok'
+labels:
+    - bchauvin
+    - mlumeau
+    - retention-management
+toc: true
+tree_item_index: 450
+---
+
+Proving compliance requires keeping a record of all the evidence you may need. Therefore, Nuxeo Server allows you to define in the most flexible way what should constitute a record. 
+
+By default, Nuxeo Server will always protect the main file of the document when [putting a document under retention]({{page page='nuxeo-retention-functional-overview'}}#put-a-document-under-retention) and the document remains editable. Should you wish to put under retention additional files from the document and prevent certain properties from being edited, you need to configure a list of properties to be protected.
+
+In this tutorial, you will learn how to leverage this configuration to retain multiple document properties securely, including how to retain additional files in WORM storage. This helps proving that no metadata has been tampered with in case of an audit. If you wish to go further, a high level understanding of how to store additional elements like comments will also be provided.
+
+## Prerequisites
+
+This feature is available starting from:
+- Nuxeo Server `LTS 2021 HF33`
+- Nuxeo Retention `2021.4.5`
```

Comment:
Aren't the fix versions one from https://jira.nuxeo.com/browse/NXP-31396 and https://jira.nuxeo.com/browse/NXP-31397 ?

Reply:
ah yes, correct! will fix that, thank you :)